src/mol-repr/volume/slice.ts

```diff
@@ -51,18 +52,44 @@ export async function createImage(ctx: VisualContext, volume: Volume, key: numbe
     const imageArray = new Uint8Array(width * height * 4);
     const groupArray = getPackedGroupArray(volume.grid, props);
 
+    const gridToCartn = Grid.getGridToCartesianTransform(volume.grid);
+    const l = PositionLocation(Vec3(), Vec3());
+
+    let v: (ix: number, iy: number, iz: number) => number;
+    if (equalEps(isoVal, stats.min, 0.001)) {
+        v = () => 255;
+    } else if (equalEps(isoVal, stats.max, 0.001)) {
+        v = () => 0;
```

Comment:
```suggestion
        v = (ix, iy, iz) => 255;
    } else if (equalEps(isoVal, stats.max, 0.001)) {
        v = (ix, iy, iz) => 0;
```

There is a de-optimization when you call a function with different than declared number of params.